packages/compile-solidity/compilerSupplier/loadingStrategies/VersionRange.js

```diff
@@ -84,7 +132,7 @@ class VersionRange extends LoadingStrategy {
     if (this.versionIsCached(versionRange)) {
       return this.getCachedSolcByVersionRange(versionRange);
     }
-    throw this.errors("noVersion", versionRange);
+    throw new NoVersionError(versionRange);
   }
 
   async getSolcByCommit(commit) {
```

Comment:
This is a preexisting, I know, but... it looks like this function isn't used anywhere.  Can it be deleted?

Reply:
I also noticed that. I wonder the provenance of this. I can eliminate.